incubator/elasticsearch/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: elasticsearch
 home: https://www.elastic.co/products/elasticsearch
-version: 1.5.0
+version: 1.5.1
```

Comment:
This is adding functionality and should therefor be a minor update which would make this version `1.6.0`.

Reply:
@yokhahn need to bump the chart version to `1.6.0` per @paulczar comments

Reply:
Should be now